client/src/components/ManageAccount/tests/ManageAccountTables.test.js

```diff
@@ -21,12 +21,12 @@ const dummyState = {
   },
   data: {
     demographics: {
-      ethnicity: 'Asian',
-      gender: 'Male',
-      bio: '',
-      website: 'http://owais.io',
-      orcid_id: 'test',
-      professional_level: 'Staff (support, administration, etc)',
+      // ethnicity: 'Asian',
```

Comment:
same here, delete this unused code

Reply:
Good question, I believe those values (on  https://accounts.tacc.utexas.edu/profile) are different than the values that we're deleting on the backend. I say this because when look at the api response on https://cep.test/accounts/api/profile/data/, those values are either empty strings or null for my profile, whereas on https://accounts.tacc.utexas.edu/profile, they are filled out. So my guess is this is separate from the PortalProfile data.